<?xml version="1.0"?>
<launch>

	<!---=================================================================================================================================================================-->
	<!-- load tool model -->
	<!-- <param name="robot_description" command="$(find xacro)/xacro '$(find ros_robot_pkg)/xacros/blade_sorting_inspection_project.urdf.xacro'" /> -->
	<!-- <param name="robot_description" command="$(find xacro)/xacro '$(find ros_robot_pkg)/xacros/blade_sorting_inspection_project.urdf.xacro'" /> -->
	<!-- <param name="robot_description" command="$(find xacro)/xacro '$(find ros_robot_pkg)/xacros/mitsubishi.urdf.xacro'" /> -->
	<!-- <param name="robot_description" command="$(find xacro)/xacro '$(find ros_robot_pkg)/xacros/ur10_laser_setup.urdf.xacro'" /> -->
	<!-- <param name="robot_description" command="$(find xacro)/xacro '$(find ros_robot_pkg)/xacros/ur10E_laser_setup.urdf_lht.xacro'" /> -->
	<param name="robot_description" command="$(find xacro)/xacro '$(find ros_robot_pkg)/xacros/ur10E_laser_setup.urdf_SANAD.xacro'" />


	<node name="robot_state_publisher" pkg="robot_state_publisher" type="robot_state_publisher" />
	<!-- <node pkg="tf2_ros" type="static_transform_publisher" name="static_transform_publisher" args="0.15 -0.9 0.4 0.0117 -0.699 0.7145 0.0192 ur_base blade" /> -->
	<!-- <node pkg="tf2_ros" type="static_transform_publisher" name="static_transform_publisher" args="0.0 -0.840 0.06 0 0 3.14 ur_base ping_pong" /> -->
   

	<!---=================================================================================================================================================================-->
	<!-- Robot Node -->
    <node pkg="ros_robot_pkg" type="ros_robot.py" name="ros_robot"  output="screen">
    </node>

</launch>
